app/build.gradle

```diff
@@ -1,10 +1,10 @@
 apply plugin: 'com.android.application'
 apply plugin: 'org.ajoberstar.grgit'
 android {
-    compileSdkVersion 32
+    compileSdk 35
+    namespace "ai.elimu.vitabu"
 
     defaultConfig {
-        applicationId "ai.elimu.vitabu"
         minSdkVersion 24
```

Comment:
@tuancoltech I think the `compileSdk` is supposed to be _inside_ the `defaultConfig{}`?